faucet.py

```diff
@@ -1,31 +1 @@
-import os
-import random
-import json
-
-from flask_qrcode import QRcode
-from flask import Flask, render_template, request
-from lightning import LightningRpc
-
-rpc = LightningRpc(os.environ['RPC_SOCKET'])
-connect_str = os.environ['CONNECT_STRING']
-app = Flask(__name__)
-QRcode(app)
-
-@app.route('/', methods=['GET', 'POST'])
-def index():
-    invoice = None
-    pay_result = None
-
-    if request.method == 'POST':
-        # create invoice
-        if 'amount' in request.form:
-            # convert to sats
-            amount = int(request.form['amount']) * 1000
-            invoice = rpc.invoice(amount, str(random.random()), 'test')['bolt11']
-        # pay invoice
-        if 'invoice' in request.form:
-            pay_result = str(rpc.pay(request.form['invoice']))
-             
-    return render_template('index.html', name='justin', 
-        invoice=invoice, pay_result=pay_result, connect_str=connect_str)
-
+from app import app
```

Comment:
Would be good to have something like the following here:

```python
if __name__ == "__main__":
    app.run(port = 3000)
```

so that it can be started without some weird other component I never got to run on Nix (I know you aren't supposed to, but don't want to waste more time on this).

Reply:
An option would be great! This is just what I did so far.